proto/src/main/proto/Rss.proto

```diff
@@ -29,6 +29,7 @@ service ShuffleServer {
   rpc sendShuffleData (SendShuffleDataRequest) returns (SendShuffleDataResponse);
   rpc getShuffleIndex (GetShuffleIndexRequest) returns (GetShuffleIndexResponse);
   rpc getShuffleData (GetShuffleDataRequest) returns (GetShuffleDataResponse);
+  rpc getInMemoryShuffleData (GetInMemoryShuffleDataRequest) returns (GetInMemoryShuffleDataResponse);
```

Comment:
Why can we reuse getShuffleData and getShuffleIndex?

Reply:
To avoid misunderstanding, will update getShuffleIndex, getShuffleData -> getLocalShuffleIndex, getLocalShuffleData